### DISCUSSION TITLE
Unmovable child node when programmatically added to group

### DISCUSSION THREAD

**Discussion by steve1two** [Blazor-Diagrams/Blazor.Diagrams#403] [Q&A]
I have a simple node that is just a title and a single port. When I call GroupModel.AddChild it adds the node fine, but I am unable to move the node. The only nodes I can move in the group are the ones that I add when I create the group.

Further, when I try to move a node that I've added after the group already exists, it will move whatever object I last moved. For instance, if I move one of the originally created nodes (on group creation) then try to move a newly added node, it moves the first node. If I go and move a different node on the screen completely unrelated to the group, then go back to the newly added group node and try to move it, it moves the other node.

I assume it has something to do with the event subscription but can't for the life of me figure out how to even debug this

**Comment by zHaytam** [comment]
Hello, can you create an issue and show me the code you're using?
